### ISSUE TITLE
Join Weights & Biases (wandb /W&B)

### ISSUE THREAD

**Issue by JoshuaPina** [gsu-ds/campus-burglary-risk-prediction#30]
### 1. What
Our WandB account has been set up for >30 days now. Harini has successfully logged runs to the project, and now we need the rest of the team fully onboarded.

We are implementing WandB to standardize how we track machine learning experiments. Currently, it is difficult to compare results across different team members' local environments. This tool solves that by moving logs from our terminals to a shared cloud dashboard.

---

### 2. Why
* **Centralize Logs:** View all training runs, loss curves, and metrics in a single shared dashboard.
* **Compare Results:** Overlay graphs from different runs to identify the best model architecture & hyperparameters.
* **Reproducibility:** Automatically track the code version, config, and hardware used for every single experiment.
* **Visualize Performance:** See rich media outputs (images/confusion matrices) live as the model trains.

---

### 3. Your Next Steps
Please complete the following checklist by **[Insert Date]**:

- [ ] **Sign Up:** Create an account at [wandb.ai](https://wandb.ai/site).
- [ ] **Join the Team:** Accept the invite to our workspace here: [Team Page](https://wandb.ai/joshuadariuspina-georgia-state-university).
- [ ] **Install locally:** Run `pip install wandb` in your terminal/environment.
- [ ] **Login locally:** Run `wandb login` in your terminal.
    * *Note: It will ask for your API key, which you can find in your user settings on the website.*
- [ ] **Gitignore Check:** Ensure `wandb/` is added to your `.gitignore` file so we don't push local log files to GitHub.
- [ ] **Test Run:** Run the `train.py` script to verify your metrics appear on the dashboard.
    * *Tip: You can stop the run after 1-2 epochs once you see the graphs updating online.*

---

### 4. Resources
* [WandB Quickstart Docs](https://docs.wandb.ai/quickstart)

**Comment by Robinker65** [comment]
I'm not sure if this link is expired or I'm getting a wrong link:

<img width="1434" height="638" alt="Image" src="https://github.com/user-attachments/assets/5e692b49-1265-4f43-b33f-a4e099a7017f" />
